### PR TITLE
feat: add ptBR (Brazilian Portuguese) in translations

### PR DESCRIPTION
**Pull request by DanSilva41**
**Changelog**

- Add translation in Portuguese (Brazil)
- Updates test suites that validate translations (*.json) and corresponding $schema.json.

### CODE REVIEW

File: packages/admin/dashboard/src/i18n/translations/ptBR.json [hunk 1]
```diff
@@ -0,0 +1,2755 @@
...
+        "receiveItems": "{{returnType}} {{id}}",
+        "restockAll": "Repor todos os itens",
+        "itemsLabel": "Itens recebidos",
+        "title": "Receber itens para #{{returnId}}",
+        "sendNotificationHint": "Notifique o cliente sobre a devolução recebida.",
+        "inventoryWarning": "Por favor, observe que ajustaremos automaticamente os níveis de inventário com base no seu input acima.",
+        "writeOffInputLabel": "Quantos itens estão danificados?",
+        "toast": {
+          "success": "Devolução recebida com sucesso.",
+          "errorLargeValue": "A quantidade é maior do que a quantidade de itens solicitada.",
```
Comment (carlos-r-l-rodrigues) [inline packages/admin/dashboard/src/i18n/translations/ptBR.json]: ### 
```suggestion
          "errorLargeValue": "A quantidade informada é maior do que a quantidade solicitada de itens.",
```

File: packages/admin/dashboard/src/i18n/translations/ptBR.json [hunk 2]
```diff
@@ -0,0 +1,2755 @@
...
+        "validation": {
+          "itemId": "Por favor, selecione um item de inventário.",
+          "quantity": "Quantidade é obrigatória. Por favor, insira um número positivo."
+        },
+        "header": "Estoque e inventário",
+        "editItemDetails": "Editar detalhes do item",
+        "manageInventoryLabel": "Gerenciar inventário",
+        "manageInventoryHint": "Quando ativado, vamos alterar a quantidade de inventário para você quando pedidos e devoluções forem criados.",
+        "allowBackordersLabel": "Permitir pedidos em atraso",
+        "allowBackordersHint": "Quando ativado, os clientes podem comprar a variante mesmo sem quantidade disponível.",
```
Comment (carlos-r-l-rodrigues) [inline packages/admin/dashboard/src/i18n/translations/ptBR.json]:
```suggestion
        "allowBackordersHint": "Quando ativado, os clientes podem comprar a variante mesmo sem estoque disponível.",
```

**Review by carlos-r-l-rodrigues — APPROVED** [review]
LGTM.
There are just minor suggestions.

Thanks for the PR @DanSilva41.

**Comment by DanSilva41** [comment]
Thanks for the notes and review, @carlos-r-l-rodrigues.

It's done!

File: packages/admin/dashboard/src/i18n/translations/__tests__/validate-translations.spec.ts [hunk 3]
```diff
@@ -38,29 +39,36 @@ function getTranslationKeys(obj: any, prefix = ""): string[] {
   return keys.sort()
 }
 
-describe("translation schema validation", () => {
-  test("en.json should have all keys defined in schema", () => {
-    const enPath = path.join(translationsDir, "en.json")
-    const enTranslations = JSON.parse(fs.readFileSync(enPath, "utf-8"))
+function validateTranslationFile(fileName: string) {
```
Comment (kasperkristensen) [inline packages/admin/dashboard/src/i18n/translations/__tests__/validate-translations.spec.ts]: Hi @DanSilva41, please undo the changes to the test file. We can't maintain translations in other languages, and this would prevent us from merging any PR's that changes the i18n schema, without also updating all the language files, which we can't.

**Review by kasperkristensen — CHANGES_REQUESTED** [review]

File: packages/admin/dashboard/src/i18n/translations/__tests__/validate-translations.spec.ts (hunk 3 above)
Comment (DanSilva41) [reply packages/admin/dashboard/src/i18n/translations/__tests__/validate-translations.spec.ts]: Understood, it makes sense, @kasperkristensen.

I'll undo.

Comment (DanSilva41) [reply packages/admin/dashboard/src/i18n/translations/__tests__/validate-translations.spec.ts]: It's done.

Thanks, @kasperkristensen!

**Review by kasperkristensen — APPROVED** [review]
LGTM! Thanks @DanSilva41

**Review by kasperkristensen — CHANGES_REQUESTED** [review]
Some new keys were added, so would be good to get them in:

Validation failed for ptBR.json:
  Missing required key: "transfer" at orders
  Missing required key: "from" at orders.activity
  Missing required key: "to" at orders.activity
  Missing required key: "transfer" at orders.activity.events

**Comment by DanSilva41** [comment]
> Some new keys were added, so would be good to get them in:
> 
> Validation failed for ptBR.json: Missing required key: "transfer" at orders Missing required key: "from" at orders.activity Missing required key: "to" at orders.activity Missing required key: "transfer" at orders.activity.events

Hi, @kasperkristensen.
It's correct now, thank you!

**Review by kasperkristensen — APPROVED** [review]
